Jenkinsfile

```diff
@@ -455,7 +455,7 @@ def tagAndPush(Map args = [:]) {
     tags << aliasVersion
   }
   // supported image flavours
-  def variants = ["", "-oss", "-ubi8"]
+  def variants = ["", "-complete", "-oss", "-ubi8"]
```

Comment:
In [this change](https://github.com/elastic/beats/pull/27427/files) I am limiting the addition of the `-complete` flavor to `elastic-agent` but here it doesn't look like you're doing that. Is there a reason?

Reply:
In https://github.com/elastic/beats/pull/27572/files/aa3181449899eb9523fb5133f40121eb7ad7dba1#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8R489 we are indicating that a Beat does not have a variant. I followed that message to always add it, although I considered too creating that logic you mention.

IIRC, maybe @v1v recalls too, when we added the `ubi8` variant, we followed the same approach: retry the `doTagAndPush` and if it always fails, consider it does not exist. Maybe it's time to make the logic a bit smarter.

Reply:
If it cleanly handles the case where a variant does not exist, that would be fine, but bear in mind that the _only_ case in which that variant should be found is with the elastic-agent. It won't be there for anything else.

Reply:
Yes, it makes sense to have it explicitly added. Will submit it in a new commit, thanks!

Reply:
Done in https://github.com/elastic/beats/pull/27572/commits/1352715d13ce343428616d727f593a77858646e3